src/Build/BackEnd/BuildManager/BuildManager.cs

```diff
@@ -3043,6 +3044,12 @@ private ILoggingService CreateLoggingService(
                 forwardingLoggers = forwardingLoggers?.Concat(forwardingLogger) ?? forwardingLogger;
             }
 
+            if (_buildParameters.EnableTargetOutputLogging)
+            {
+                loggingService.EnableTargetOutputLogging = true;
+            }
```

Comment:
There's _a lot_ of this kind of forwarding of configuration across project/buildmanager/buildparameters/etc. It's a little noisy but hopefully mostly straightforward.